user_data.sh

```diff
@@ -61,14 +62,20 @@ ${name}\:db-import-from-s3:
 	pv $(TMP_DIR)/$(DUMP_BASENAME).sql.gz | gzip -dc | sudo mysql --defaults-file=$(MY_CNF) $(DB)
 	@echo "Create additional databases..."
 	find $(TMP_DIR) -name "*.gz" -printf "%f\n" | \
-	  sed -e "s/\..*$///" | \
-	  sed -e "s/$(DUMP_BASENAME)//" | \
-	  xargs -I '{}' sudo mysql --defaults-file=$(MY_CNF) -e "CREATE DATABASE IF NOT EXISTS $(DB){}"
+		sed -e "s/\..*$///" | \
```

Comment:
suggest using `'....'` instead of `"..."` because in this notation `$/` is somewhat ambiguous. It should be a bash error, but it's probably ignored.